lib/elastomer/version.rb

```diff
@@ -1,5 +1,5 @@
 module Elastomer
-  VERSION = "3.0.1"
+  VERSION = "3.0.2"
```

Comment:
I think this is generally better to do in a separate commit, so we can bump the version for a few changes at once. 

You probably had to in order to publish it locally but I wouldn't check it in.